frontend/javascripts/oxalis/controller/viewmodes/arbitrary_controller.js

```diff
@@ -77,6 +77,10 @@ class ArbitraryController extends React.PureComponent<Props> {
   listenTo: Function;
   stopListening: Function;
 
+  static defaultProps = {
+    onRender: () => {},
```

Comment:
I think, `onRender` was only used by the fps meter. So, the prop can be removed completely from my point of view (same for plane_controller).